asm/rom_header.s

```diff
@@ -17,7 +17,7 @@
 .ascii INTERNAL_ROM_NAME   /* Internal ROM name */
 .word  0x00000000               /* Unknown */
 .word  0x0000004E               /* Cartridge */
-.ascii "ED"                     /* Cartridge ID */
```

Comment:
This should be behind a define. For people who don't care about VC and are using an alternate save type besides EEP4k, ED can be used to indicate to some flashcarts and emulators what save type to use for the game.

Reply:
Fixed by specifying "SM" only for EEP4K but keep "ED" for any other type. Let's avoid VC defines alltogether because it is nearly impossible to hack decomp hacks If I will want to port to VC (no one cares about VC basically).